src/Panels/ChatLogPanel.cs

```diff
@@ -397,6 +414,9 @@ private static void PrintMessage(string msg)
                     {
                         chatPanel.isVisible = true;
                         chatPanel.Update();
+
+                        // Reset the timeout counter
+                        chatPanel._timeoutCounter = 0;
```

Comment:
Since this is in the if with `!chatPanel.isVisible`, it will not reset the timer when the window is already open.
Maybe restructure the if, so that the visible check is in a separate if only for the two lines above.